deploy/ubuntu/control

```diff
@@ -0,0 +1,26 @@
+Source: edumfa
+Maintainer: eduMFA <edumfa-dev@listserv.dfn.de>
+Section: python
+Priority: optional
+Build-Depends: python3-all (>= 3.5), debhelper (>= 7.4.3), dh-virtualenv (>= 0.11), quilt, virtualenv, lsb-release, git
+Standards-Version: 3.9.5
+
+Package: edumfa
+Architecture: amd64
+Replaces: python-privacyidea
+Breaks: python-privacyidea
+Conflicts: edumfa-venv, edumfa-appliance (<= 2.1)
```

Comment:
I think edumfa-venv and edumfa-appliance should be removed since that never existed?

Reply:
So should we remove this completely or change it to privacyidea?

Reply:
you can change it to privacyidea-*, doesn't hurt to conflict with these

Reply:
@j-hoff I just did a test run. `dpkg-gencontrol: warning: can't parse dependency privacyidea-*`

Reply:
yes, no wildcards allowed. sorry, it was only my abbreviation :)